directories/src/redux/services/iota.js

```diff
@@ -1,19 +1,48 @@
 import IOTA from "iota.lib.js";
-import { IOTA_API_PROVIDER } from "../../config";
+import { IOTA_API_PROVIDER, IOTA_ADDRESS_LENGTH } from "../../config";
 import curl from "curl.lib.js";
+import _ from "lodash";
+import moment from "moment";
 
 const iota = new IOTA();
 
-const providerIota = new IOTA({
+const iotaProvider = new IOTA({
   provider: IOTA_API_PROVIDER
 });
 
 curl.init();
 const MAX_TIMESTAMP_VALUE = (Math.pow(3, 27) - 1) / 2;
 
+const checkIfClaimed = address =>
+  findTransactions([address]).then(transactions => {
+    const latestTransaction = transactions[0];
```

Comment:
I am not entirely certain that iota will always return them in any particular chronological order.  Even if they did, I think the most recent one would be the last one, not the 0th one?  

@automyr I'm thinking you will have some input on this, you were talking about this issue in Slack today.

Reply:
Indeed `findTransactions` does not return them in chronological order (at least going by timestamps). 

Jet and I are working on an IXI command to check and compare timestamps.